continuous_integration/environment-3.13.yaml

```
@@ -0,0 +1,55 @@
+name: dask-distributed-313
+channels:
+  - conda-forge
+dependencies:
+  - python=3.13
+  - packaging
+  - pip
+  - asyncssh
+  - bokeh>3
+  - click
+  - cloudpickle
+  - coverage
+  - dask  # overridden by git tip below
+  - filesystem-spec  # overridden by git tip below
+  - gilknocker
+  - h5py
+  - ipykernel
+  - ipywidgets
+  - jinja2
+  - jupyter-server-proxy
+  - jupyterlab
+  - locket
+  - msgpack-python
+  - netcdf4
+  - paramiko
+  - pre-commit
+  - prometheus_client
+  - psutil
+  #- pyarrow
+  - pytest
+  - pytest-cov
+  - pytest-faulthandler
+  - pytest-repeat
+  - pytest-rerunfailures
+  - pytest-timeout
+  - requests
+  - s3fs  # overridden by git tip below
+  - scikit-learn
+  - scipy
+  - sortedcollections
+  - tblib
+  - toolz
+  - tornado
+  - zict  # overridden by git tip below
+  - zstandard
+  # Temporary fix for https://github.com/pypa/setuptools/issues/4496
+  - setuptools < 71
+  - pip:
+      - git+https://github.com/dask/dask
+      # - git+https://github.com/dask-contrib/dask-expr
```

I'm not sure why this is commented out. @phofl is this still needed?

```suggestion
      - git+https://github.com/dask/dask-expr
```

---

dask-expr pulls in pyarrow which wasn't compatible with python 3.13

---

Ah, I see. `pyarrow=18` was recently released and has Python 3.13 support, so I think we should be okay to add this back in then

---

Yes definitely, I was waiting for that